README.md

```diff
@@ -183,7 +183,7 @@ inventory = $HOME/cloudera-playbook/dynamic_inventory_cm_py2
 # Python 3 version:
 # inventory = $HOME/cloudera-playbook/dynamic_inventory_cm_py3
 # Do not gather the host information (facts) by default. This can give significant speedups for large clusters.
-gathering = explicit
+# gathering = explicit
 # Disable key check if host is not initially in 'known_hosts'
 host_key_checking = False
 [ssh_connection]
```

Comment:
why so much diff'ed lines shown?
(maybe LF vs CRLF issues?!)

Reply:
Hi @lhoss,

Thanks for the review! I do not know why do we have this strange view in github. If you open  this, it is a bit different view template:

https://github.com/cloudera/cloudera-playbook/pull/58/files/c10a800609a64e7aeaf5e24661169fe255099457?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8